### ISSUE TITLE
a way to have "swipe-right" calls back the "previous card"

### ISSUE THREAD

**Issue by futurk** [swaplet/react-native-swipe-cards-deck#27]
i think it's possible through changing the code but it would be so convenient if there was a shortcut option (e.g. prop) to achieve this.

**Comment by eyalabs** [comment]
Do you mean like a gallery, swipe left right for navigation?

**Comment by futurk** [comment]
> Do you mean like a gallery, swipe left right for navigation?

yes. actually i came up with some kind of workaround. this modification worked for me:
in the file SwipeCards.js, at the row 158;
instead of `if (status.finished) this._advanceState();`
this:
```
if (status.finished)
  if (hasMovedLeft) this._advanceState();
  else if (hasMovedRight) this._goToPrevCard();
```

**Comment by eyalabs** [comment]
This wasn't intended for this library, but if you already did the modifications, I'll look at them later to make it an integral part. 
Thank you for your contribution.
